static/app/views/issueDetails/groupCronChecks.tsx

```diff
@@ -0,0 +1,267 @@
+import {Fragment} from 'react';
+import {useTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+import moment from 'moment-timezone';
+
+import Duration from 'sentry/components/duration';
+import GridEditable, {type GridColumnOrder} from 'sentry/components/gridEditable';
+import LoadingError from 'sentry/components/loadingError';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {Tooltip} from 'sentry/components/tooltip';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {User} from 'sentry/types/user';
+import {defined} from 'sentry/utils';
+import {FIELD_FORMATTERS} from 'sentry/utils/discover/fieldRenderers';
+import parseLinkHeader from 'sentry/utils/parseLinkHeader';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useParams} from 'sentry/utils/useParams';
+import {useUser} from 'sentry/utils/useUser';
+import {EventListTable} from 'sentry/views/issueDetails/streamline/eventListTable';
+import {useCronIssueAlertId} from 'sentry/views/issueDetails/streamline/issueCronCheckTimeline';
+import {useGroup} from 'sentry/views/issueDetails/useGroup';
+import {type CheckIn, CheckInStatus} from 'sentry/views/monitors/types';
+import {statusToText, tickStyle} from 'sentry/views/monitors/utils';
+import {scheduleAsText} from 'sentry/views/monitors/utils/scheduleAsText';
+import {useMonitorChecks} from 'sentry/views/monitors/utils/useMonitorChecks';
+
+export default function GroupCronChecks() {
```

Comment:
```suggestion
export default function GroupCronCheckIns() {
```

Reply:
Changed everything to just `CheckIns` for now 👍